static/app/views/performance/queues/messageSamplesPanel.tsx

```diff
@@ -139,16 +152,23 @@ export function MessageConsumerSamplesPanel() {
         <ModuleLayout.Layout>
           <ModuleLayout.Full>
             <HeaderContainer>
-              {project && (
+              {project ? (
                 <SpanSummaryProjectAvatar
                   project={project}
                   direction="left"
                   size={40}
                   hasTooltip
                   tooltip={project.slug}
                 />
+              ) : (
+                <div />
```

Comment:
`null` also fine here

Reply:
I think we actually need an element here because the parent container is a `grid` and the formatting gets messed up without an element:
![image](https://github.com/getsentry/sentry/assets/83961295/4cbe8e2f-4c4f-4fdb-91c9-ee7b241f05a9)

Reply:
Oof, okay!